app/src/main/java/io/intrepid/contest/screens/contestcreation/describecontest/DescribeContestPresenter.java

```diff
@@ -14,15 +13,30 @@
     }
 
     @Override
+    public void onViewCreated() {
+        super.onViewCreated();
+        view.disableNext();
+    }
+
+    @Override
     public void onNextClicked(String description) {
```

Comment:
this may actually not be needed now that the user can't progress without filling in the information. Ask the designers what the protocol is.